py3-hatchling is currently in community/ while py3-pygments is in main/.
An aport in main/ cannot depend upon one in community/, so build with
py3-setuptools while waiting for py3-hatchling to be moved to main/.


--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,7 @@
 [build-system]
-requires = ["hatchling"]
-build-backend = "hatchling.build"
+# setuptools added pyproject.toml support in v61.0.0
+requires = ["setuptools >= 61"]
+build-backend = "setuptools.build_meta"
 
 [project]
 name = "Pygments"
@@ -53,12 +54,11 @@
 [project.scripts]
 pygmentize = "pygments.cmdline:main"
 
-# Temporary workaround for https://github.com/pypa/hatch/issues/1054
-[tool.hatch.build.targets.wheel]
-packages = ["pygments"]
+[tool.setuptools.dynamic]
+version = {attr = "pygments.__version__" }
 
-[tool.hatch.version]
-path = "pygments/__init__.py"
+[tool.setuptools.packages.find]
+include = ["pygments", "pygments.*"]
